corehq/form_processor/models.py

```diff
@@ -536,8 +536,9 @@ def dynamic_case_properties(self):
 
     def to_json(self):
         from .serializers import CommCareCaseSQLSerializer
-        serializer = CommCareCaseSQLSerializer(self)
-        return serializer.data
+        data = CommCareCaseSQLSerializer(self).data
+        data.update(self.case_json)
+        return data
```

Comment:
cool that's more compact than [my version](https://github.com/dimagi/commcare-hq/commit/17e339b0662e769375d1af31a096d6ecbb82869c). I was worried about clobbering the global namespace